contrib/remotexact/remotexact.c

```diff
@@ -261,6 +261,7 @@ rx_send_rwset_and_wait(void)
 	header = &rwset_collection_buffer->header;
 	pq_sendint32(&buf, header->dbid);
 	pq_sendint32(&buf, header->xid);
+	pq_sendint64(&buf, header->csn);
```

Comment:
Could you also update the line that sends csn in `rx_collect_page` to use `pq_sendint64`?

Reply:
Done.